pkg/verify/verify.go

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package verify
```

Comment:
seems like we're going to need to write some tests for this (or copy them?)

Reply:
Some of it can probably be inferred from https://github.com/sigstore/rekor/blob/main/pkg/verify/verify_test.go but the types are going to be different